src/test/java/org/moditect/jfrunit/JfrUnitTest.java

```diff
@@ -66,7 +72,7 @@ public void shouldHaveGcAndSleepEventsWithDefaultConfiguration() throws Exceptio
                 event("jdk.ThreadSleep").with("time", Duration.ofMillis(50)));
 
         assertThat(jfrEvents.filter(
-                event("jdk.GarbageCollection").with("cause", "System.gc()")))
+                event("jdk.GarbageCollection").with("cause", "System.gc()")).collect(Collectors.toList()))
```

Comment:
Why does it need the `collect()` call now?

Reply:
This was included when I squashed some commits. I will remove it. However, I need to investigate this further.  I was seeing the `shouldHaveGcAndSleepEventsWithDefaultConfiguration` fail as there were 2 GC events, but only one with the cause "System.gc()". There was another GC event, but the cause was "G1 Evacuation Pause"  The filter was not filtering out the second GC and the assertion was failing.